src/app/lib/helpers.ts

```diff
@@ -51,8 +51,13 @@ export function concat(...parts: Uint8Array[]) {
   return result
 }
 
+/**
+ * Parse a ROSE (or TEST) amount from a string.
+ *
+ * An empty string will be interpreted as 0.
+ */
 export function parseRoseStringToBaseUnitString(value: string): StringifiedBigInt {
-  const baseUnitBN = new BigNumber(value).shiftedBy(9) // * 10 ** 9
+  const baseUnitBN = new BigNumber(value || '0').shiftedBy(9) // * 10 ** 9
```

Comment:
This function shouldn't be responsible for handling this

Reply:
> I am not exactly sure what are expectations here. Do we really don't want to catch an error when a function input is invalid? Why we even trigger a submit event when a form is empty?

@buberdds your concerns have been addressed in the new implementation. (We no longer submit the form when the field is empty.)